Sources/Extensions/Foundation/StringExtensions.swift

```diff
@@ -18,4 +18,14 @@ public extension String {
 
         return ceil(boundingBox.width)
     }
+
+    func withLineSpacing(_ space: CGFloat) -> NSAttributedString {
```

Comment:
Should it perhaps be called `attributedStringWithLineSpacing ` since it creates `NSAttributedString`?

Reply:
I agree!